forest/src/main.rs

```diff
@@ -6,23 +6,118 @@ mod daemon;
 mod logger;
 mod subcommand;
 
-use cli::{cli_error_and_die, Cli};
+use cli::{cli_error_and_die, Cli, DaemonConfig};
+
+use async_std::task;
+use daemonize_me::{Daemon, DaemonError, Group, User};
+use log::info;
+use raw_sync::events::{Event, EventInit};
+use raw_sync::Timeout;
+use shared_memory::ShmemConf;
 use structopt::StructOpt;
 
-#[async_std::main]
-async fn main() {
+use std::fs::File;
+use std::process;
+use std::time::Duration;
+
+const EVENT_TIMEOUT: Timeout = Timeout::Val(Duration::from_secs(20));
```

Comment:
It would be nice to have this in config instead. Not a big deal though.

Reply:
I think this should be ok for now.